docs/SMCWG-charter.md

```diff
@@ -90,9 +90,6 @@ root, limited as follows:
   be present)
 - CA operational practices, physical/logical security, etc.
 
-The SMCWG needs to exercise caution to ensure that its work product does not 
-impede the issuance of certificates with other EKU types.
```

Comment:
Avoiding conflicts is a good idea, but trying to enforce it via vague language in the charter is probably not helpful.